flang/include/flang/Optimizer/Dialect/CUF/CUFOps.td

```diff
@@ -161,10 +161,11 @@ def cuf_DataTransferOp : cuf_Op<"data_transfer", []> {
 
   let arguments = (ins Arg<AnyType, "", [MemRead]>:$src,
                        Arg<AnyRefOrBoxType, "", [MemWrite]>:$dst,
+                       Variadic<AnyIntegerType>:$shape,
```

Comment:
Is there a reason for not using a `!fir.shape` value here?

Reply:
No special reason. I took the definition for the AllocOp but shape is fine. I updated the op.